interfaces/repo.go

```diff
@@ -332,6 +334,57 @@ func (r *Repository) ResolveConnect(plugSnapName, plugName, slotSnapName, slotNa
 	return ref, nil
 }
 
+// ResolveDisconnect resolves potentially missing plug or slot names and
+// returns a list of fully populated connection references that can be
+// disconnected.
+func (r *Repository) ResolveDisconnect(plugSnapName, plugName, slotSnapName, slotName string) ([]ConnRef, error) {
```

Comment:
It is, indirectly, with connect/disconnect tests, but you are right. I will add dedicated tests for it.